static/templates/compose.hbs

```diff
@@ -79,9 +79,9 @@
                                 </span>
                                 <a role="button" class="narrow_to_compose_recipients zulip-icon zulip-icon-arrow-left-circle order-1" data-tooltip-template-id="narrow_to_compose_recipients_tooltip" tabindex="0">
                                 </a>
-                                <input type="text" class="recipient_box" name="stream_message_recipient_stream" id="stream_message_recipient_stream" maxlength="30" value="" placeholder="{{t 'Stream' }}" autocomplete="off" tabindex="0" aria-label="{{t 'Stream' }}" />
+                                <input type="text" class="recipient_box" name="stream_message_recipient_stream" id="stream_message_recipient_stream" maxlength="{{ max_stream_name_length }}" value="" placeholder="{{t 'Stream' }}" autocomplete="off" tabindex="0" aria-label="{{t 'Stream' }}" />
```

Comment:
The value for `max_stream_name_length` seems to be 60 as of now, but the previous value was `30`. I'm not sure why there was this inconsistency before. I think this should be fine, but just wanted to highlight the change.

Reply:
@jai2201 this is the sort of detail that should absolutely be discussed explicitly in the commit message. I added that to the commit message before merging this, but any anomaly like this is important to highlight for reviewers, as regressions are often introduced through reviewers being unaware of a functional change like this in what would appear to be a refactoring commit.